types/yallist/yallist-tests.ts

```diff
@@ -92,6 +92,8 @@ myList.sliceReverse(); // $ExpectType Yallist<string>
 myList.sliceReverse(0); // $ExpectType Yallist<string>
 myList.sliceReverse(1, -2); // $ExpectType Yallist<string>
 
+myList.splice(0, 0); // $ExpectType string[]
```

Comment:
Would you mind adding a second test for the optional parameter?

Reply:
Sure :)